.gitignore

```diff
@@ -76,6 +76,7 @@ db.sqlite3-journal
 virtualenv
 docker-compose.yml
 docker-compose.env
+.vscode/
```

Comment:
As this isn't project-specific, I'd suggest that you add `.vscode` to your personal git ignore file (on Linux systems, this is at `${HOME}/.config/git/ignore`).

Reply:
Oh, I totally missed that - thanks! Rebase is on the way.